gbpservice/nfp/config_orchestrator/agent/loadbalancerv2.py

```diff
@@ -0,0 +1,457 @@
+    def delete_listener(self, context, listener):
+        loadbalancer = listener['loadbalancer']
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(loadbalancer["description"])
+        nf = common.get_network_function_details(context, nf_id)
+        self._delete(
+            context, listener['tenant_id'],
+            'listener', nf, listener=listener)
+
+    @log_helpers.log_method_call
+    def create_pool(self, context, pool):
+        loadbalancer = pool['loadbalancer']
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(loadbalancer["description"])
+        nf = common.get_network_function_details(context, nf_id)
+        self._post(
+            context, pool['tenant_id'],
+            'pool', nf, pool=pool)
+
+    @log_helpers.log_method_call
+    def delete_pool(self, context, pool):
+        loadbalancer = pool['loadbalancer']
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(loadbalancer["description"])
+        nf = common.get_network_function_details(context, nf_id)
+        self._delete(
+            context, pool['tenant_id'],
+            'pool', nf, pool=pool)
+
+    @log_helpers.log_method_call
+    def create_member(self, context, member):
+        loadbalancer = member['pool']['loadbalancer']
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(loadbalancer["description"])
+        nf = common.get_network_function_details(context, nf_id)
+        self._post(
+            context, member['tenant_id'],
+            'member', nf, member=member)
+
+    @log_helpers.log_method_call
+    def delete_member(self, context, member):
+        loadbalancer = member['pool']['loadbalancer']
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(loadbalancer["description"])
+        nf = common.get_network_function_details(context, nf_id)
+        self._delete(
+            context, member['tenant_id'],
+            'member', nf, member=member)
+
+    @log_helpers.log_method_call
+    def create_healthmonitor_on_pool(self, context, pool_id, healthmonitor):
+        loadbalancer = healthmonitor['pool']['loadbalancer']
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(loadbalancer["description"])
+        nf = common.get_network_function_details(context, nf_id)
+        self._post(
+            context, healthmonitor['tenant_id'],
+            'healthmonitor', nf, healthmonitor=healthmonitor)
+
+    @log_helpers.log_method_call
+    def create_healthmonitor(self, context, healthmonitor):
+        loadbalancer = healthmonitor['pool']['loadbalancer']
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(loadbalancer["description"])
```

Comment:
Both create_healthmonitor_on_pool and create_healthmonitor has same code. Retain the actual API.